### DISCUSSION TITLE
Is this a bug? (2 folder with the same name)

### DISCUSSION THREAD

**Discussion by RallyTuning** [community/community#43989] [Repositories] [answered]
### Select Topic Area

Bug

### Body

Hello all, i have this repository in my account and i use it with Visual Studio 2022:
![image](https://user-images.githubusercontent.com/50233545/211865902-cf15c0b5-2c67-45e3-ad70-afa619b5f287.png)
As you can see, there are 2 folders with the SAME name but one have the "n" not capital.
On my computer (Windows) i see just ONE folder (the one with N capital) that contain 2 files `project.csproj` and `Starter.cs`.
But here on GitHub i see that one folder contain the csproj file (n) and the other one contain the Starter.cs (N).
How i can merge them? If i clean all the files and make a new pull request, i can see again ONE folder on my computer but still TWO on GitHub repository.
Thank you

**Comment by RallyTuning** [comment, marked as answer]
Solved by renaming the folder on Windows, push the commits and the renaming it back to normal.

**Comment by simaotwx** [comment]
That's because NTFS on Windows is case-insensitive, meaning that upper and lower case names refer to the same file or folder.
So git has no way of distinguishing the two when using Windows and NTFS which means that the folders appear merged.

**Reply by RallyTuning** [reply]
Yes, i have read that before trying the fix that i wrote before. Looking in my commits, this happened when i renamed the class from `class.cs` to `Starter.cs`, Visual Studio also rename the folder with the capital N, i pushed the commits and then the folder split on GitHub. Now, i know that this is normal in Linux and in this context, but for who use Windows, will cause this "problem". Just saying
